### DISCUSSION TITLE
Scene-viewer shadow

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4873] [Q&A] [answered]
So i'm using wall placement in AR and i set **shadow-intensity="0"** and **shadow-softness="0"** but when defaulting to scene-viewer instead of webXR, the shadow is always visible on the wall... Do we have any control over shadows in Scene-Viewer?

Also, we can't change light estimation in Scene-Viewer right?

**Comment by elalish** [comment, marked as answer]
No, there is almost no control over either Scene Viewer or Quick Look, as they are separate apps outside the browser with very narrow APIs for configuration. Only WebXR mode gives you that kind of control.
